### PR TITLE
Make sure vexflow can be loaded with jspm

### PR DESCRIPTION
**Pull request by khisakuni**
This allows tsc to be able to find the module when loaded with jspm

### CODE REVIEW

File: vexflow/vexflow.d.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@
 //inconsistent namespace: this is a helper funtion from tables.js and should not pollute the global namespace!
 declare function sanitizeDuration(duration : string) : string;
 
-declare namespace Vex {
+declare module Vex {
```
Comment (vvakame) [inline vexflow/vexflow.d.ts]: why did you need this change?

Comment (khisakuni) [inline vexflow/vexflow.d.ts]: I think it needs to be declared as a `module` for jspm to load it. I went off the lodash package as a reference.

**Comment by rquiring** [comment]
I don't know much about jspm so I might be missing something here, but I think this should be integrated into VexFlow itself rather than the VexFlow Typescript definitions. Have a look at the universal module definitions at the bottom of Vex.js. Also VexFlow already supports npm, so from what I understand jspm should just work out of the box?

File: vexflow/vexflow.d.ts (hunk 1 above)
Comment (vvakame) [inline vexflow/vexflow.d.ts]: You should use `namespace` instead of internal `module`.
https://github.com/Microsoft/TypeScript/issues/2159

**Comment by khisakuni** [comment]
@vvakame Sorry, I'm new to TS so I was unaware of the preferred namespace syntax. It's updated now.

**Comment by khisakuni** [comment]
@rquiring Now that I've thought more about it, I don't think it has to do with jspm at all. I'm still new to TypeScript so I apologize if I'm missing something here, but doesn't the `d.ts` file have to export something? Otherwise when I try to do `import * as Vex from 'vexflow'`, the compiler complains that it can't find the module. I see in other typescript definitions what I'm trying to do here. (E.g. [lodash](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/lodash/lodash.d.ts), [axios](https://github.com/khisakuni/DefinitelyTyped/blob/master/axios/axios.d.ts), [fastclick](https://github.com/khisakuni/DefinitelyTyped/blob/master/fastclick/fastclick.d.ts)

**Comment by vvakame** [comment]
LGTM.

https://github.com/0xfe/vexflow/blob/dc32d0f4f767c87b62062342a4a1d36c6522bc53/releases/vexflow-debug.js#L194-L202
vexflow exports modules by UMD pattern.
